composer/modules/web/src/plugins/ballerina/graphical-editor/controllers/lifeline.jsx

```diff
@@ -315,7 +315,7 @@ class ActionBox extends React.Component {
         const { viewState: { bBox } } = model.getBody();
 
         const top = model.viewState.components.defaultWorker.y + 20;
-        const left = bBox.x;
+        const left = bBox.x + 12;
```

Comment:
Can we use a constant here instead of a magic number?

Reply:
Fixed with commit 8b45d50